argilla/mkdocs.yml

```diff
@@ -142,7 +142,7 @@ nav:
       - Query and filter records: how_to_guides/query.md
       - Importing and exporting datasets: how_to_guides/import_export.md
       - Annotate a dataset: how_to_guides/annotate.md
-      - Migrate your legacy datasets to Argilla V2: how_to_guides/migrate_from_legacy_datasets.md
+      - Distribute the annotation: how_to_guides/distribution.md
```

Comment:
I'm not sure if the title is understandable at first glance, maybe adding "task" is better.